packages/atomic/src/components/common/search-box/submit-button.spec.ts

```diff
@@ -0,0 +1,72 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {createTestI18n} from '@/vitest-utils/testing-helpers/i18n-utils';
+import {userEvent} from '@storybook/test';
+import {html} from 'lit';
+import {describe, vi, beforeAll, it} from 'vitest';
+import {expect} from 'vitest';
+import {renderSubmitButton} from './submit-button';
+
+describe('#renderSubmitButton', () => {
+  let i18n: Awaited<ReturnType<typeof createTestI18n>>;
+
+  beforeAll(async () => {
+    i18n = await createTestI18n();
+  });
+
+  const renderComponent = (additionalProps = {}) => {
+    return renderFunctionFixture(
+      html`${renderSubmitButton({
+        props: {
+          i18n,
+          ...additionalProps,
+        },
+      })}`
+    );
+  };
+
+  it('should have the "submit-button-wrapper" part on the wrapper', async () => {
+    const element = await renderComponent();
+    const wrapper = element.querySelector('div');
+    expect(wrapper).toHaveAttribute('part', 'submit-button-wrapper');
```

Comment:
See my comment in this other PR here: https://github.com/coveo/ui-kit/pull/5249/files#r2071721982